build.gradle

```diff
@@ -53,7 +53,7 @@ subprojects {
 
     ext {
         mesosVer = "0.24.1"
-        hadoopVer = "2.7.0"
+        hadoopVer = "2.7.2"
```

Comment:
We should have this as property probably so not to change build.gradle every time